styles/components/_formErrors.scss

```diff
@@ -0,0 +1,21 @@
+.errorList {
+  color: $error;
+  
+  li {
+    i {
+      margin-right: 5px;
+    }
+
+    margin-bottom: 5px;
+  }
+
+  margin-bottom: 10px;
+}
+
+label {
+  &.error {
+    input, textarea {
+      border: 1px solid red;
```

Comment:
This might get weird with the way radio buttons have the label inside the button. But we can fix that later after everything's merged. We'll need to get a design for errors from @morebrownies.